tests/metrics.test.ts

```diff
@@ -284,26 +286,83 @@ describe('getMoveDistanceSingleGame', () => {
       moves: game,
     });
 
-    expect(result.maxDistancePiece).toBe('pe');
+    expect(result.maxDistancePiece).toBe('PE');
```

Comment:
which is black and which is white

Reply:
white capitalized, black lower case. realizing that chess.js had it opposite, which is confusing bc it's white capitalized in the chess world

Reply:
so which are doing, now i am confused

Reply:
right now code for unambig pieces between chess.ts and what we have written is structured for white to be capitalized, no need to change anything unless prefer otherwise

getMateAndAssists function also working properly for ambig pieces, haven't tested other functions or other functions aren't using ambig pieces